### ISSUE TITLE
Some style attributes don't parse correctly

### ISSUE THREAD

**Issue by cjke** [goopscoop/svg-to-react-cli#7]
Script trips on certain styles: 

Input: 
```
<svg viewBox="0 0 93.87 436.96">
    <rect style="fill:#ccc" />
</svg>
```

Output (note the misplaced #):
```
import React from 'react';

export default function WineBottle({width = '50px', height = '50px'}) {
  return (
    <svg viewBox="0 0 93.87 436.96" height={height} width={width}>
  <rect style={{fill:#"cc}} />
</svg>


  );
}

WineBottle.propTypes = {
  width: React.PropTypes.string,
  height: React.PropTypes.string
}
```

Happy to submit a PR. 
Could consider this: https://github.com/harrison/react-magic/tree/add-html-svg-attributes to do the heavy lifting. I've seen it used quite well here: https://github.com/gocardless/splash-pages/blob/dev/script/generators/render-icons

**Comment by goopscoop** [comment]
Thanks for catching that. A PR would be great. If that project produces a more consistent output, then I'm all for it.

I should probably add unit tests to this project in the near future so we can catch these issues easier.

**Comment by cjke** [comment]
Great I will look at a PR over the next few days 👍

**Comment by cjke** [comment]
This is closed with PR #9
